source/blog/2016-07-14-new-centos-atomic-host-releases-available-for-download.html.md

```diff
@@ -0,0 +1,45 @@
+---
+title: New CentOS Atomic Host Releases Available for Download
+author: jbrooks
+date: 2016-07-14 07:00:00 UTC
+tags: docker, kubernetes, centos
+published: false
+comments: true
+---
+
+Last week, the [CentOS Atomic SIG](http://wiki.centos.org/SpecialInterestGroup/Atomic) [released](https://seven.centos.org/2016/07/new-centos-atomic-host-ready-for-download/) an updated version of CentOS Atomic Host (tree version 7.20160707), featuring updated versions of docker and the atomic run tool. 
+
+CentOS Atomic Host includes these core component versions:
+
+* docker-1.10.3-44.el7.centos.x86_64
+* kubernetes-1.2.0-0.12.gita4463d9.el7.x86_64
+* kernel-3.10.0-327.22.2.el7.x86_64
+* atomic-1.10.5-5.el7.x86_64
+* flannel-0.5.3-9.el7.x86_64
+* ostree-2016.5-3.atomic.el7.x86_64
+* etcd-2.2.5-2.el7.0.1.x86_64
+* cloud-init-0.7.5-10.el7.centos.1.x86_64
+
+CentOS Atomic Host is available as a VirtualBox or libvirt-formatted Vagrant box, or as an installable ISO, qcow2 or Amazon Machine image. Check out the [CentOS wiki](https://wiki.centos.org/SpecialInterestGroup/Atomic/Download) for download links and installation instructions, or read on to learn more about what's new in this release.
+
+## OCI Advances: runC and OCI Hooks
+CentOS Atomic Host now ships with [runC](http://runc.io/), a lightweight client wrapper around libcontainer for spawning and running containers according to the [Open Container Initiative](https://www.opencontainers.org/) specification. To learn about what you can do with runC and OCI, check out [this post](http://www.projectatomic.io/blog/2016/04/running_OCI/) from [Mrunal Patel](https://twitter.com/mrunalp), and this [Dockercon 2016 talk](https://www.youtube.com/watch?v=ZAhzoz2zJj8) from [Phil Estes](https://twitter.com/estesp).
+
+Users who wish to run systemd inside of their containers can do so more simply with this new atomic host release, thanks to a pair of [OCI hooks](https://github.com/projectatomic/oci-systemd-hook) that enable users to run systemd in docker and OCI compatible runtimes such as runc without requiring the `--privileged` flag, and to display journal information from these containers using the host's journalctl command.
+
```

Comment:
Wasn't runc added because of the new system container work in atomic? That feature might be worth a mention here as well.